### PR TITLE
feat: add UriParamExtension

### PR DESCRIPTION
**Pull request by loong-y**

### CODE REVIEW

File: services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/entityselector/EntitySelectorComponent.java [hunk 1]
```diff
@@ -28,6 +29,7 @@ public class EntitySelectorComponent implements ProcessorNode<Exchange> {
 
     @OutputArguments
     @UriParam(javaType = "java.util.List")
```
Comment (leonside) [inline services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/entityselector/EntitySelectorComponent.java:31]: miss  bean prefix?

File: services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/webhook/WebhookComponent.java [hunk 2]
```diff
@@ -24,10 +25,9 @@
 public class WebhookComponent implements ProcessorNode<Exchange> {
 
     @UriParam
```
Comment (leonside) [inline services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/webhook/WebhookComponent.java:27]: miss bean prefix

File: services/workflow/workflow-service/src/main/resources/camel-schema/trigger.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
   },
   "properties": {
     "name": { "index": 0, "kind": "path", "displayName": "Name", "group": "common", "label": "", "required": true, "type": "string", "javaType": "java.lang.String", "deprecated": false, "deprecationNote": "", "autowired": true, "secret": false, "description": "Name of direct endpoint" },
-    "entityConfigs": {
+    "ParamDefineInput": {
```
Comment (leonside) [inline services/workflow/workflow-service/src/main/resources/camel-schema/trigger.json:34]: Add the uiComponent parameter

File: services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/entityselector/EntitySelectorComponent.java (hunk 1 above)
Comment (loong-y) [reply services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/entityselector/EntitySelectorComponent.java:31]: Fixed

File: services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/webhook/WebhookComponent.java (hunk 2 above)
Comment (loong-y) [reply services/workflow/workflow-service/src/main/java/com/milesight/beaveriot/rule/components/webhook/WebhookComponent.java:27]: Fixed

File: services/workflow/workflow-service/src/main/resources/camel-schema/trigger.json (hunk 3 above)
Comment (loong-y) [reply services/workflow/workflow-service/src/main/resources/camel-schema/trigger.json:34]: fixed

**Pull request merged**